nautobot/core/apps/__init__.py

```diff
@@ -20,8 +22,25 @@
 
 
 logger = logging.getLogger(__name__)
+
 registry["nav_menu"] = {"tabs": {}}
 registry["homepage_layout"] = {"panels": {}}
+registry["system_jobs"] = []
+
+
+def register_jobs(class_list):
+    """
+    Register a list of Job classes
+    """
+    from nautobot.extras.jobs import Job
+
+    for job in class_list:
+        if not inspect.isclass(job):
+            raise TypeError(f"Job class {job} was passed as an instance!")
+        if not issubclass(job, Job):
+            raise TypeError(f"{job} is not a subclass of extras.jobs.Job!")
+
+        registry["system_jobs"].append(job)
```

Comment:
This was directly lifted from `nautobot.extras.plugins.utils`. I'm thinking this should be centralized as util.

Reply:
Is any of this redundant with `register_jobs` in the base branch (https://github.com/nautobot/nautobot/pull/3483/files#diff-e4b3fe39da892dc97291f6a19ae5449758cd5a1353cb61a19cbc6bffe41ade4eR182-R185)?

Reply:
Kind of? I think we're trying to converge them. For the moment, the above code is for registering the job classes in Nautobot, and the one you linked is registering them in Celery.

Reply:
We are going to unify all Jobs to be registered in the same way, and that most likely means just using the Celery task registry as the registry and reworking the other APIs to pull from that.